packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -310,10 +311,23 @@ class BottomNavigationBar extends StatefulWidget {
   /// Whether the labels are shown for the selected [BottomNavigationBarItem].
   final bool? showSelectedLabels;
 
+  /// {@template flutter.material.bottomnavigationbar.mouseCursor}
```

Comment:
This API doc can't really be re-used for the theme property because the types of the properties are different.